### ISSUE TITLE
RPC tests fail on Buildbot

### ISSUE THREAD

**Issue by defuse** [Electric-Coin-Company/zcash#691]
http://ci.leastauthority.com:8010/builders/zcashrpc/builds/0/steps/rpc-tests.py/logs/stdio

Strange error:

```
Upon execvpe ./qa/pull-tester/rpc-tests.py ['./qa/pull-tester/rpc-tests.py'] in environment id 140289403270040
:Traceback (most recent call last):
  File "/home/bbslave/venv/local/lib/python2.7/site-packages/twisted/internet/process.py", line 424, in _fork
    environment)
  File "/home/bbslave/venv/local/lib/python2.7/site-packages/twisted/internet/process.py", line 502, in _execChild
    os.execvpe(executable, args, environment)
  File "/home/bbslave/venv/lib/python2.7/os.py", line 355, in execvpe
    _execvpe(file, args, env)
  File "/home/bbslave/venv/lib/python2.7/os.py", line 370, in _execvpe
    func(file, *argrest)
OSError: [Errno 2] No such file or directory
```

**Comment by defuse** [comment]
Oh oops, I think it should just be .sh instead of .py.

**Comment by defuse** [comment]
Yeah, it's running now.

**Comment by ebfull** [comment]
The RPC tests are still failing but due to another issue I'm currently looking into...

**Comment by ebfull** [comment]
I fixed the tests in #698.

**Comment by defuse** [comment]
That's merged. Closing.
